public/js/maps.js

```diff
@@ -0,0 +1,54 @@
+const HOUSTON_COORDS = { lat: 29.7604, lng: -95.3698 }
```

Comment:
Nit: no leading/trailing spaces for inline object constants, also add a semicolon to the end.